napari/components/viewer_model.py

```diff
@@ -234,7 +234,7 @@ def _update_viewer_grid(self):
     @validator('theme')
     def _valid_theme(cls, v):
         themes = available_themes()
-        if v not in available_themes():
+        if not is_theme_available(v):
             raise ValueError(
                 trans._(
                     "Theme '{theme_name}' not found; options are {themes}.",
```

Comment:
Probably not a big deal, but the call to `available_themes()` can be moved to after the if now.

Reply:
fixed